### ISSUE TITLE
Ht keep playing the first note only

### ISSUE THREAD

**Issue by DiarioMental** [utz82/HoustonTracker2#37]
I install ht2 in my ti-83+ recently, I made the tutorial, but when I hit play, the program keep playing the first note only, the same when I put new notes, keeps playing that note only, like a drone thing and stop when I hit the play button (ENTER)

I re-install the program 2 times and still happen.
I have DoorsCS (last version) and ht2s.8xp

**Comment by utz82** [comment]
Hard to say. Things that could possibly be the cause:

- Song speed is set to a very high value (= very slow speed)
- HOLD is activated (CLEAR key)
- Not all the positions on the current row of the sequence are filled in

**Comment by DiarioMental** [comment]
Well, I tried this 3 option and don't work.
-I set the speed to 11, 30 and 59 and still the same.
-when I press the hold button not affect the progression, but shutdown my calculator. I tried this finishing the tutorial, when I hear the first note freezes, and before start making any sound. In this third case, the calculator shutdown when I put the first note.

**Comment by DiarioMental** [comment]
Oh, and the third one I filled complete the current row and put something in the next one too.

**Comment by utz82** [comment]
Hm, that's very weird. Especially that the hold button shuts down your 
calculator. It sounds like the RAM of your calc might be corrupt. Did 
you do a full reset before installing HT2?

Could you check the ROM version of your calc with this key sequence: 
[2nd] [MEM] [1]
(Don't press ENTER afterwards, it will enter self-test and clear memory. 
I think it's [2nd] [MODE] or [CLEAR] to get out.

Also, please check if installing ht2s.8xp instead of ht2.8xp helps.

**Comment by DiarioMental** [comment]
I think I made a full reset, yes.
My calculator is TI-83 Plus, 1.18

I'm gonna try it when I have the PC (I have to use an old 486 with windows 95 of a friend to install everything)

**Comment by utz82** [comment]
Alright. If possible, also pull a backup from the calculator with TiLP 
(with HT2 installed) and post it here, so I can run some tests.

**Comment by DiarioMental** [comment]
Sadly TiLP doesn't work with my PC/calculator, I use the previous version of TIConnect, if I can make a back up with that, sure I send you that :)

**Comment by DiarioMental** [comment]
Well, after like 7 months finally I fix this.
I install the last version of the OS (1.19) and reset the TI, sadly I can't make a back up because TiLP doesn't work in the PC I use, and TI graphlink and connect doesn't give me the option to make a back up.
What I think can be, is the previous OS version I used (1.18) can be incompatible with ht2, or just need another software reset maybe? IDK

Somebody who use ht2 said if you have any problem using the 1.18 version of the OS.

**Comment by utz82** [comment]
Thanks for the follow-up! Will keep an eye on OS 1.18.

**Comment by DiarioMental** [comment]
Well, I have the same problem for a mistake I made.
Trying to use too much bpm fx in a song accidentally crash the program and come to the same problem. BUT, i reset the calculator, re-install all and works ok.

I made this new comment because now I was able to make a back-up of the calculator now to share. I hope this can be useful.
[backup.zip](https://github.com/utz82/HoustonTracker2/files/11637884/backup.zip)
